package.json

```diff
@@ -900,20 +900,24 @@
     "vscode:prepublish": "npm run compile",
     "compile": "tsc -p ./",
     "lint": "eslint src --ext .ts",
-    "watch": "tsc -watch -p ./"
+    "watch": "tsc -watch -p ./",
+    "pretest": "npm run compile",
+    "test": "node ./out/test/runTest.js"
   },
   "dependencies": {
-    "tmp": "0.0.31",
+    "tmp": "0.2.1",
     "vscode-nls": "^2.0.1"
   },
   "devDependencies": {
+    "@types/mocha": "^7.0.2",
```

Comment:
This is 7.0.2 and mocha is 7.2.0
Is this intentional?

Reply:
I don't fully understand the whole `@types` thing in npm, but my sense is that if the type defs in a module don't change between versions, they don't cut a new @types version of that module. The most recent version of `@types/mocha` is 7.0.2, so hopefully that's good enough for code that uses `mocha` 7.2.0. I included both modules because that's what the sample extension does.